packages/opentelemetry-instrumentation-openai/tests/traces/test_chat.py

```diff
@@ -33,6 +56,44 @@ def test_chat(exporter, openai_client):
     assert open_ai_span.attributes.get(SpanAttributes.LLM_IS_STREAMING) is False
 
 
+@pytest.mark.vcr
+def test_chat_tools(exporter, openai_client, openai_tools):
+    openai_client.chat.completions.create(
+        model="gpt-3.5-turbo",
+        messages=[
+            {"role": "user", "content": "What's the weather like in San Francisco?"}
+        ],
+        tools=openai_tools,
+    )
+
+    spans = exporter.get_finished_spans()
+
+    assert [span.name for span in spans] == [
```

Comment:
These tests are redundant for the 3 cases (non-streaming, streaming, async streaming). Should I use a shared helper function here for the `assert` statements?

Reply:
No, it's better to be explicit in tests, even if it's repetitive. Maybe we can switch to a parameterized test (but not important for this PR)